.github/workflows/pr-build.yaml

```diff
@@ -1,14 +1,18 @@
-name: Pull request
+name: Pull Request - Builds
 
 on: pull_request
 
 env:
   TURBO_SCM_BASE: ${{ github.event.pull_request.base.sha }}
   TURBO_SCM_HEAD: ${{ github.sha }}
 
+concurrency:
+  group: push
+  cancel-in-progress: false
+
 jobs:
-  test:
-    name: Test-build with Node.js ${{ matrix.node-version }}
+  build:
+    name: Build with Node.js ${{ matrix.node-version }}
```

Comment:
this change will affect the prow config at https://github.com/openshift/release/blob/master/core-services/prow/02_config/janus-idp/backstage-plugins/_prowconfig.yaml#L12